src/App.tsx

```diff
@@ -15,25 +15,36 @@ export function App(): JSX.Element | null {
       <TermContext.Provider value={{ term, setTerm }}>
         <Flex h="100vh" direction="column">
           <Header />
-          <Box grow={1} overflow="hidden" height="100%">
-            <Flex color="white" height="100%">
-              <SidebarContainer term={term as Term} pid={pid || ''} setPid={setPid} />
-              <Flex minW="80%" overflow="auto" justifyContent="center" height="100%" boxShadow="lg" zIndex={56}>
-                {pid ? (
-                  <Content term={term as Term} pid={pid} />
-                ) : (
-                  <Center p="10">
-                    <VStack>
-                      <Heading color="black">Explore Courses</Heading>
-                      <Text color="gray">
-                        Select a subject and then a course to start viewing course details and section information.
-                      </Text>
-                    </VStack>
-                  </Center>
-                )}
-              </Flex>
-            </Flex>
-          </Box>
+          <Router>
+            <Routes>
+              <Route path="/" />
+              <Route
+                path="/calendar"
+                element={
+                  <Box grow={1} overflow="hidden" height="100%">
+                    <Flex color="white" height="100%">
+                      <SidebarContainer term={term as Term} pid={pid || ''} setPid={setPid} />
+                      <Flex minW="80%" overflow="auto" justifyContent="center" height="100%" boxShadow="lg" zIndex={56}>
+                        {pid ? (
+                          <Content term={term as Term} pid={pid} />
+                        ) : (
+                          <Center p="10">
+                            <VStack>
+                              <Heading color="black">Explore Courses</Heading>
+                              <Text color="gray">
+                                Select a subject and then a course to start viewing course details and section
+                                information.
+                              </Text>
+                            </VStack>
+                          </Center>
+                        )}
+                      </Flex>
+                    </Flex>
+                  </Box>
+                }
+              />
```

Comment:
@keithradford should we move this code into a `pages` folder?

Reply:
agreed, or `routes` folder?

Reply:
yeah, I think `routes` makes more sense given they aren't necessarily "pages". Either way I think a directory is justified